scripts/build.sh

```diff
@@ -322,11 +323,17 @@ do
         BINARY="$OUTFILE.exe"
     fi
 
+    DYNAMIC_LDFLAGS=$LDFLAGS
+    if [[ "$GOARM" != "" ]]
+    then
+        DYNAMIC_LDFLAGS = "$DYNAMIC_LDFLAGS -X github.com/stellar/kelp/cmd.goarm=$GOARM"
+    fi
+
     gen_bundler_json -p $GOOS
     gen_bind_files
     # compile
     echo -n "compiling for (GOOS=$GOOS, GOARCH=$GOARCH, GOARM=$GOARM) ... "
-    env GOOS=$GOOS GOARCH=$GOARCH GOARM=$GOARM go build -ldflags "$LDFLAGS" -o $BINARY
+    env GOOS=$GOOS GOARCH=$GOARCH GOARM=$GOARM go build -ldflags "$DYNAMICLDFLAGS" -o $BINARY
```

Comment:
`DYNAMIC_LDFLAGS` instead of `DYNAMICLDFLAGS`

I feel like this should have been caught by testing out the build script.

Reply:
Good catch.  These are exactly the changes made during our pair session on Monday, and the script ran successfully then - so not sure why that worked.